met-web/src/components/common/index.tsx

```diff
@@ -179,34 +180,96 @@ export const modalStyle = {
     color: Palette.text.primary,
 };
 
-export const MetHeader1 = styled(Typography)(() => ({
-    fontSize: '2.3rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
-
-export const MetHeader2 = styled(Typography)(() => ({
-    fontSize: '1.9rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
-
-export const MetHeader3 = styled(Typography)(() => ({
-    fontSize: '1.5rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
-
-export const MetHeader4 = styled(Typography)(() => ({
-    fontSize: '1.3rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
+interface HeaderProps {
+    sx?: SxProps;
+    bold?: boolean;
+    children?: React.ReactNode | string;
+    align?: 'left' | 'right' | 'inherit' | 'center' | 'justify';
+    style?: React.CSSProperties;
+    onClick?: () => void;
+    color?: string;
+}
+export const MetHeader1 = ({ bold, children, sx, onClick, color, align, style }: HeaderProps) => {
+    return (
+        <Typography
+            sx={{
+                ...sx,
+                fontSize: '2.3rem',
+                fontWeight: bold ? 'bold' : MET_Header_Font_Weight,
+                fontFamily: MET_Header_Font_Family,
+            }}
+            style={{ ...style }}
```

Comment:
export const SecondaryButton = ({ children, ...rest }: { children: React.ReactNode; [prop: string]: unknown }) => (
    <StyledSecondaryButton {...rest} variant="outlined">
        {children}
    </StyledSecondaryButton>
);

please check the 'rest' prop above and apply it to the Headers. You then don't have to have the onClick, color, align etc. as props. They will all be part of rest. Perhaps maybe the sx keep since you're overriding it and bold since it is not originally in Typography.

Reply:
Rest literally means whatever other props are passed